package/MDAnalysis/coordinates/XTC.py

```diff
@@ -64,10 +64,16 @@ def __init__(self, filename, n_atoms, convert_units=True,
             number of atoms to write
         convert_units : bool (optional)
             convert into MDAnalysis units
+        dt : float (optional)
+            timestep in MDAnalysis units to load trajectory with
+            If dt is None, time for a frame is set from the timestep
```

Comment:
same comment on formatting

Reply:
Please check that you use "to load trajectory with" for **readers** and "to write trajectory with" for **writers**. I think I saw some instances where this just seemed copied and pasted across.